src/vs/editor/common/config/editorOptions.ts

```diff
@@ -980,7 +984,7 @@ class EditorAccessibilitySupport extends BaseEditorOption<EditorOption.accessibi
 	}
 }
 
-//#endregion
+
```

Comment:
maybe this was deleted by accident, or was there an extra `//#endregion` in the file?

Reply:
By accident. My mistake.